fix(docs): content review (#2131)
scaleway/docs-content: dedibox-network/rpn/reference-content/private-networks-overview.mdx

> When deploying infrastructure on the Internet, network connectivity will play a crucial part in your success in terms of both performance and security. You want your services to run quickly, smoothly and with minimum latency, while also ensuring that sensitive data is fully protected even as it is being transferred over a public network like the Internet.  

Reviewer: ```suggestion
When deploying infrastructure on the Internet, network connectivity plays a crucial part in your success in terms of both performance and security. You want your services to run quickly, smoothly and with minimum latency, while also ensuring that sensitive data is fully protected even as it is being transferred over a public network like the Internet.  
```

​

The Internet was initially designed to enable a few academics and researchers to share data between themselves. As such, security was not a paramount concern. Of course, over time the Internet grew, with more and more private companies and personal users taking it up, until today it has become the omnipresent global network we all know, linking billions of machines and people worldwide. While technologies have evolved and data-protection mechanisms to secure our connections have become more sophisticated, we should never forget that the Internet remains essentially a public network. This means that infrastructure attacks like Distributed Denial-of-Service or Brute force can always stand to pose a threat to your infrastructure and your data.  
​

> Scaleway provides a range of solutions to help you protect against such attacks and secure your infrastructure. Besides DDoS protection, we also offer several ways to create private networks, fully compliant with [RFC 1918](https://docs.umbrella.com/umbrella-user-guide/docs/internal-networks-setup-guide). These networks can add extra protection for your data, since they are entirely isolated, and completely separate from the Internet or any public network.

Reviewer: ```suggestion
Scaleway provides a range of solutions to help protect you against such attacks and secure your infrastructure. Besides DDoS protection, we also offer several ways to create private networks, fully compliant with [RFC 1918](https://docs.umbrella.com/umbrella-user-guide/docs/internal-networks-setup-guide). These networks can add extra protection for your data, since they are entirely isolated, and completely separate from the Internet or any public network.
```

To interconnect your Scaleway Dedibox dedicated servers and your Scaleway Instances, you can choose to configure a **VPN**: a **V**irtual **P**rivate **N**etwork. Here, the public Internet interface is used to create an anonymous and encrypted private network. In essence, a VPN creates an encrypted tunnel between you and the internet, routing all your internet traffic through this tunnel and masking your IP address and data as it travels. Since your internet activity is hidden by the VPN, you have enhanced privacy and anonymity. With VPNs, you can create hybrid infrastructures across all of Scaleway's product ranges, even if Instances or servers are not in the same availability (geographic) zone.

> You could, for example, configure a Dedibox dedicated server as a load-balancing front end that forwards traffic through a VPN to virtual Instances which serve the requests of your users, allowing you to scale your application depending on the load. These virtual Instances can then communicate over the VPN with a database, running on an Elastic Metal Server for performance and reliability. Endless options are possible and can be realized using techniques such as a [WireGuard VPN](/tutorials/wireguard-mesh-vpn/) or using a [GRE Tunnel](/tutorials/setting-up-tunnel/).

Reviewer: ```suggestion
You could, for example, configure a Dedibox dedicated server as a load-balancing front end that forwards traffic through a VPN to virtual Instances which serve the requests of your users, allowing you to scale your application depending on the load. These virtual Instances can then communicate over the VPN with a database, running on an Elastic Metal server for performance and reliability. Endless options are possible and can be realized using techniques such as a [WireGuard VPN](/tutorials/wireguard-mesh-vpn/) or using a [GRE Tunnel](/tutorials/setting-up-tunnel/).
```
